requirements.txt

```diff
@@ -0,0 +1,31 @@
+alembic==0.8.9
+aniso8601==1.2.0
+click==6.6
+colorama==0.3.7
+coverage==4.3.1
+coveralls==1.1
+docopt==0.6.2
+Flask==0.12
+Flask-HTTPAuth==3.2.1
+Flask-Login==0.4.0
+Flask-Migrate==2.0.2
+Flask-RESTful==0.3.5
+Flask-Script==2.0.5
+Flask-SQLAlchemy==2.1
+Flask-Testing==0.6.1
+itsdangerous==0.24
+Jinja2==2.8.1
+Mako==1.0.6
+MarkupSafe==0.23
+nose==1.3.7
+passlib==1.7.0
```

Comment:
passlib and Werkzeug do the same thing... `from werkzeug.security import generate_password_hash, check_password_hash` It would be better to just use Werkzeug as it comes with a Flask installation.